### PR TITLE
Add an acronym list

### LINKED ISSUE #48: Add a glossary
There may be a library-wide one we can use

Otherwise, need to lookup acronyms used in this department.

A list we can add to:
- PRDS: Princeton Research Data Service, part of the Office of Research Data and Open Scholarship
- There are a lot of projects we might add here, e.g. "Geniza"
- DPSG / DPOG: Digital Projects Steering group, changing to Digital Projects Operations Group. This group vets and approved new digitization projects, which are a primary source of new feature requests for Figgy and DPUL.
- ReCAP: [The Research Collections and Preservation  Consortium](https://recap.princeton.edu/) ReCAP refers to both the consortium and its offsite storage location that stores materials from Princeton, NYPL, and Columbia (and soon Harvard)
- SCSB: Shared Collection Service Bus, pronounced "scuzzy-B", middleware used by ReCAP to integrate their inventory system with our ILS / bibdata
- LAS: The ReCAP inventory system
- HTC: The contract company that develops ReCAP's inventory system
- CDL: controlled digital lending. An orangelight / figgy feature and also a white paper
- OIT: Office of Information Technology, the university's central IT department
- SN@P: Service Now at Princeton, the issue tracker and knowledge base management system used by OIT and by the library operations team.
- OARSC: Open Access to Restricted Special Collections - this is the functionality provided by figgy that allows us to make restricted items available digitally to specific people. Related to the "Virtual Reading Room" project
- POD: platform for open discovery. A "data lake" of marc data from all 13 Ivies+ institutions

### PR DESCRIPTION
**Pull request by hackartisan**
I looked through the centrally-managed list and pulled in a couple that seem
frequently used by our team. Combined with the list that was in the issue, I
think this is pretty comprehensive. We can continue adding to it.

fixes #48

### CODE REVIEW

File: acronyms.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+This list is for acronyms you'll especially hear in DRDS and closely related teams. There is also a list of terms and acronyms managed by the organizational effectiveness office. There may be some overlap.
+
+The [list of committees](https://pulstaff.princeton.edu/pul-committees/) might be helpful, as well.
+
+* ADAPT (Archival Description and Processing Team): The archivists' team
+* CaMS (Cataloging and Metadata Services): The catalogers' team
+* CDL (controlled digital lending): An orangelight / figgy feature and also a white paper
+* DPSG / DPOG (Digital Projects Steering group, changing to Digital Projects Operations Group): This group vets and approved new digitization projects, which are a primary source of new feature requests for Figgy and DPUL.
+* DSSG (Discovery Services Steering group): The group that advises development of orangelight
+* HTC: The contract company that develops ReCAP's inventory system
```
Comment (christinach) [inline acronyms.md]: I think DSSG is in general for discovery and not only for orangelight.

Comment (hackartisan) [reply acronyms.md]: In practice the other discovery apps have different product owners.

Comment (christinach) [reply acronyms.md]: But this is the file for the acronyms not for the product owners.

Comment (christinach) [reply acronyms.md]: https://pulstaff.princeton.edu/discovery-services-steering-group-dssg/

**Review by kelynch — APPROVED** [review]

File: acronyms.md (hunk 1 above)
Comment (hackartisan) [reply acronyms.md]: @christinach I updated it

**Review by christinach — APPROVED** [review]

**Review by christinach — APPROVED** [review]

**Pull request merged**